app/src/main/java/com/linecorp/linesdktest/MainActivity.java

```diff
@@ -47,10 +47,12 @@ protected void onCreate(Bundle savedInstanceState) {
         setContentView(R.layout.activity_main);
         getWindow().setSoftInputMode(WindowManager.LayoutParams.SOFT_INPUT_STATE_ALWAYS_HIDDEN);
 
-        getSupportFragmentManager()
-                .beginTransaction()
-                .add(R.id.container, new MenuFragment())
-                .commit();
+        if (getSupportFragmentManager().getFragments().isEmpty()) {
```

Comment:
When Activity is re-created, and fragment manager is not empty, there's no need to create a new MenuFragment.